nodejs/Cargo.toml

```diff
@@ -0,0 +1,25 @@
+[package]
+edition = "2021"
+name = "autonomi-nodejs"
+version = "0.1.0"
+description = "NodeJS bindings for the autonomi client"
+license = "GPL-3.0"
+
+[lib]
+crate-type = ["cdylib"]
+
+[dependencies]
+autonomi = { path = "../autonomi" }
+bls = { package = "blsttc", version = "8.0.1" }
```

Comment:
To avoid compatibility across versions, I would suggest we remove this dependency and re-export whatever we need from bls in autonomi.

Reply:
Good point, will refactor that